admin/src/main/resources/io/buoyant/admin/js/src/router_summary.js

```diff
@@ -94,14 +93,10 @@ define([
         var m = metrics[metric];
         var datum = _(data).get(m.accessor);
 
-        if(m.isPath) {
-          return _.get(datum, m.metricAccessor) || 0;
-        } else {
-          return _.reduce(datum, function(mem, entityData) {
-            mem += _.get(entityData, m.metricAccessor) || 0;
-            return mem;
-          }, 0);
-        }
+        return _.reduce(datum, function(mem, entityData) {
+          mem += _.get(entityData, m.metricAccessor) || 0;
+          return mem;
```

Comment:
nit, and realize this was already here, but, should this be `value` or `stat` or something, not `mem`?